docs/UserGuide.md

```diff
@@ -173,75 +202,104 @@ Format: `give INDEX d/DESCRIPTION by/ D/M/YYYY [,HHMM]`
 
 Displays the list of people who match any of the input keywords. The matching is based
 on an ***OR*** basis, where if a student matches at **least one keyword**, that student will
 be considered as matched and thus displayed.
 
-Format: `find [n/NAME] [m/MODULE] [t/TUTORIAL_NUMBER]...`
+Format: `find [n/NAME] [m/MODULE] [t/TAG]...`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+
+1. There should be at least one prefix.
+
+2. Ordering of prefixes are not strict and presence of multiple keywords are acceptable.
 
-**Note:**
-1. At least one prefix is required.
-2. Ordering of prefixes are not strict, and allows for multiple keywords
 3. Keywords are **case-insensitive** <br> e.g. `cs1101s` will match `CS1101S`.
-4. Partial names will be matched. <br> e.g. `n/Hans` will match `Hans Bo`.
-5. If there are two identical prefixes in the command, only the last one works.
-6. Students' names matching at least one keyword will be returned (i.e. `OR` search). <br>
+
+4. Each part of name separated by space will match the full name. <br> e.g. `n/Hans` or `n/Bo` will both match `Hans Bo`.
+
+5. Students' names matching at least one keyword will be returned (i.e. `OR` search). <br>
    e.g. `n/Hans Bo` will return `Hans Gruber`, `Bo Yang`.
-7. We can search for multiple fields, e.g., modules at once, separated by a whitespace. <br>
-   e.g. `m/CS1101S CS2103T` will return people who take either / or both modules.
+
+</div>
+
+<div markdown="span" class="alert alert-primary">
+
+:bulb: **Tip:**
+* We can search for multiple fields, e.g., modules at once, separated by a whitespace. 
+   e.g. `m/CS1101S CS2103T` will return people who take either modules.
+</div>
 
 Examples:
-* `find n/Bernice` returns the students with name `Bernice`. 
+* `find n/Bernice` returns the students with name of `Bernice`. 
 * `find m/CS1101S` returns the students with module `CS1101S`.
 * `find t/Lab15` returns the students with tutorial `Lab15`.
-* `find n/Bernice m/MA1521` returns the students with name `Bernice` or study module `MA1521`.
+* `find n/Bernice m/MA1521` returns the students with name of `Bernice` or study module `MA1521`.
 * `find m/CM1417 t/Group04` returns the students with module `CM1417` or in tutorial `Group04`.
-* `find n/Evian m/CS2103T t/Group10` returns the students who at least satisfy one of requirements: with name `Evian`, study module`CS2103T` and in tutorial `Group10`.
-* `find t/E34 n/Brian m/GEQ1000` returns the students who at least satisfy one of requirements: with name `Brian`, study module`GEQ1000` and in tutorial `E34`.
-* `find n/alex david m/cs1231 cs2103t` returns the students who at least satisfy one of requirements: with name contains `david` or `alex`, study module`cs1231` or `cs2103t`.
+* `find n/Evian m/CS2103T t/Group10` returns the students who at least satisfy one of the requirements: with name of `Evian`, study module`CS2103T` and in tutorial `Group10`.
```

Comment:
"and in tutorial" should be replaced with "or with the tag `Group10`

Reply:
good catch,done